pkg/pillar/dpcreconciler/genericitems/dhcpcd.go

```diff
@@ -26,6 +25,8 @@ import (
 const (
 	dhcpcdStartTimeout = 3 * time.Second
 	dhcpcdStopTimeout  = 30 * time.Second
+
+	zeroIPv4Addr = "0.0.0.0"
```

Comment:
What about IPv6?

Reply:
@uncleDecart we currently don't have logic to drive dhcpv6 for the ports. We use SLAAC with IPv6.